backend/app/router/endpoints/questions.py

```
@@ -2,48 +2,80 @@
-            ],
-            correct_answer=0
-        ),
-        Question(
-            question="Which theorem connects continuous symmetries to conservation laws in physics?",
-            answer="Noether's theorem"
-        )
-    ])
+    load_dotenv(dotenv_path=os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))), '.env'))
+
+    client = OpenAI(
+    api_key=os.environ.get("LLAMA_API_KEY"), 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)